api_core/google/api_core/client_info.py

```diff
@@ -92,5 +92,6 @@ def to_user_agent(self):
 
         if self.client_library_version is not None:
             ua += "gccl/{client_library_version} "
+            ua += "gcloud-python/{client_library_version} "
```

Comment:
Why would we want to repeat the same version with a different prefix?

Reply:
The gccl header is the way forward, we have systems that still expect gcloud-python. This is to make sure both legacy and forward systems have data in the form they expect. I will mark this dnm for now.

Reply:
ISTM we need to double-check that adding `gcloud-python` back in doesn't break any of the log parsers expecting the "standard" format.

Reply:
It shouldn't be an issue due to the additive nature of the ua strings. That said, I am currently in discussions with some folks.